gframe/data_manager.cpp

```diff
@@ -343,6 +344,16 @@ const CardDataC* DataManager::GetMappedCardData(uint32_t code) const {
 		return gDataManager->GetCardData(it->second);
 	return nullptr;
 }
+std::string DataManager::GetNameString(uint32_t code) const {
+	std::wstring name;
+	auto csit = cards.find(code);
+	if(csit == cards.end() || csit->second.GetStrings().name.empty())
+		name = {};
+	name = csit->second.GetStrings().name;
+
+	std::wstring_convert<std::codecvt_utf8<wchar_t>, wchar_t> converter;
+	return converter.to_bytes(name);
+}
```

Comment:
I don't think there's the need for a separate function, `GetName` should be enough for this usecase, at worst you'd get the `???` when an unknown card is saved (thing only possible if a deck is saved while repos are still being loaded).